js/twitterFetcher.js

```diff
@@ -285,6 +285,7 @@
             tweet: tweets[n].innerHTML,
             author: authors[n].innerHTML,
             time: times[n].textContent,
+            timestamp: times[n].getAttribute('datetime').replace('+0000', 'Z').replace(/([\+\-])(\d\d)(\d\d)/, '$1$2:$3'),
```

Comment:
Can you explain what this line is doing exactly?  Specifically: .replace(/([\+\-])(\d\d)(\d\d)/, '$1$2:$3')

From what I see once you have replaced +0000 with Z  you get:

"2016-09-21T16:45:42Z"

And then the 2nd replace doesnt do anything... This is certainly not a timestamp that most people would be thinking of eg epoch. Can you clarify what you are achieving here?

Would it not be easier to do:

timestamp: new Date(times[n].getAttribute('datetime')).getTime(),

Which in the example above would produce:

1474476342000